lib/components/molecules/Quote/Quote.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+import React from 'react';
+import type { Node } from 'react';
+import styled from 'styled-components';
+import classnames from 'classnames';
+import styles from './Quote.style';
+
+type Props = {
+  className: string,
+  content: string,
+  contentSrc: string,
+  author?: string,
+  company?: string,
+};
+
+const Quote = ({ content, contentSrc, author, company, className, ...otherProps }: Props): Node => {
+  return (
+    <div {...otherProps} className={classnames('quote-wrap', className)}>
```

Comment:
Make the block-quote outside most component. Examples below 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote 
http://html5doctor.com/blockquote-q-cite/